css/style.css

```diff
@@ -76,3 +80,11 @@ th {
     width: 100%;
     margin-bottom: 10px;
 }
+
+h5{
+    display:none;
+    visibility:hidden;
```

Comment:
Could we add a custom class or something to this? Making an unguarded `display: none` on a standard style seems dangerous to me.